Description: Macros for user paths. Based on Marcelo E. Magallon <mmagallo@debian.org> diff patch.
Author: Rodolfo García Peñas (kix) <kix@kix.es>
Last-Update: 2012-01-09
--- a/Workspace/wmmacros
+++ b/Workspace/wmmacros
@@ -22,7 +22,7 @@
 /*
  * SETSTYLE FILE - loads a style file 
  */
-#define SETSTYLE EXEC cd $HOME/.workspace/Library/Workspace/Styles; setstyle 
+#define SETSTYLE EXEC cd $GNUSTEP_USER_ROOT/Library/Workspace/Styles; setstyle
 
 
 /*
@@ -60,6 +60,14 @@
 #define STYLES_DIR		#wmdatadir#/Styles
 #define ICON_SETS_DIR		#wmdatadir#/IconSets
 #define BACKGROUNDS_DIR		#wmdatadir#/Backgrounds
+#define LOCAL_THEMES_DIR	/usr/local/share/Workspace/Themes
+#define LOCAL_STYLES_DIR	/usr/local/share/Workspace/Styles
+#define LOCAL_ICON_SETS_DIR	/usr/local/share/Workspace/IconSets
+#define LOCAL_BACKGROUNDS_DIR	/usr/local/share/Workspace/Backgrounds
+#define USER_THEMES_DIR	$GNUSTEP_USER_ROOT/Library/Workspace/Themes
+#define USER_STYLES_DIR	$GNUSTEP_USER_ROOT/Library/Workspace/Styles
+#define USER_ICON_SETS_DIR	$GNUSTEP_USER_ROOT/Library/Workspace/IconSets
+#define USER_BACKGROUNDS_DIR	$GNUSTEP_USER_ROOT/Library/Workspace/Backgrounds
 
 
 #define StaticGray		0
